core/src/main/java/jenkins/util/AntClassLoader.java

```diff
@@ -902,9 +906,7 @@ public URL getResource(final String name) {
         if (url != null) {
             log("Resource " + name + " loaded from parent loader", Project.MSG_DEBUG);
         } else {
-            // try and load from this loader if the parent either didn't find
-            // it or wasn't consulted.
-            url = getUrl(pathComponents, name);
+            url = getUrl(name);
```

Comment:
I cleaned up and upstreamed this change in apache/ant#150. This just adjusts our local copy to match the final version that was accepted upstream.